src/Entity/Aid.php

```diff
@@ -0,0 +1,464 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\FundraisingCardRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Mapping\Annotation as Gedmo;
+use Symfony\Bridge\Doctrine\IdGenerator\UuidV4Generator;
+use Symfony\Component\Uid\UuidV4;
+
+/**
+ * @Gedmo\SoftDeleteable(fieldName="deletedAt", timeAware=false, hardDelete=false)
+ * @ORM\Entity(repositoryClass=FundraisingCardRepository::class)
+ * @ORM\Table(name="aids")
+ */
+class Aid
+{
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="uuid", unique=true)
+     * @ORM\GeneratedValue(strategy="CUSTOM")
+     * @ORM\CustomIdGenerator(class=UuidV4Generator::class)
+     */
+    private UuidV4 $id;
+
+    /**
+     * @Gedmo\Timestampable(on="create")
+     * @ORM\Column(type="datetime")
+     */
+    private \DateTimeInterface $createdAt;
```

Comment:
You can put that in a trait if you wish, I usually create `src/Entity/Util/EntityTimestampable.php` and `src/Entity/Util/EntitySoftDeletable.php`